addons/godot-xr-tools/functions/function_pose_area.gd

```diff
@@ -0,0 +1,108 @@
+class_name XRToolsFunctionPoseArea, "res://addons/godot-xr-tools/editor/icons/hand.svg"
```

Comment:
I'm wondering about this name, I know we have an area within but it doesn't really communicate what the class is for. What about `XRToolsFunctionPoseDetector` ?

Reply:
Renamed to XRToolsFunctionPoseDetector